languages/states.php

```diff
@@ -0,0 +1,4429 @@
+		'KA' => __( 'Muḩāfaz̧at Karbalā’', 'lifterlms' ),
+		'MU' => __( 'Muḩāfaz̧at al Muthanná', 'lifterlms' ),
+		'QA' => __( 'Muḩāfaz̧at al Qādisīyah', 'lifterlms' ),
+		'NI' => __( 'Nineveh', 'lifterlms' ),
+		'SD' => __( 'Salah ad Din', 'lifterlms' ),
+		'SU' => __( 'Sulaymaniyah', 'lifterlms' ),
+	),
+	'IR' => array(
+		'10' => __( 'Isfahan', 'lifterlms' ),
+		'11' => __( 'Sistan and Baluchestan', 'lifterlms' ),
+		'12' => __( 'Ostān-e Kordestān', 'lifterlms' ),
+		'13' => __( 'Ostān-e Hamadān', 'lifterlms' ),
+		'14' => __( 'Chaharmahal and Bakhtiari', 'lifterlms' ),
+		'15' => __( 'Ostān-e Lorestān', 'lifterlms' ),
+		'16' => __( 'Ilam Province', 'lifterlms' ),
+		'17' => __( 'Kohgiluyeh and Boyer-Ahmad', 'lifterlms' ),
+		'18' => __( 'Bushehr', 'lifterlms' ),
+		'19' => __( 'Zanjan', 'lifterlms' ),
+		'20' => __( 'Semnan', 'lifterlms' ),
+		'21' => __( 'Yazd', 'lifterlms' ),
+		'22' => __( 'Hormozgan', 'lifterlms' ),
+		'23' => __( 'Ostān-e Tehrān', 'lifterlms' ),
+		'24' => __( 'Ostān-e Ardabīl', 'lifterlms' ),
+		'25' => __( 'Qom', 'lifterlms' ),
+		'26' => __( 'Ostān-e Qazvīn', 'lifterlms' ),
+		'27' => __( 'Golestan', 'lifterlms' ),
+		'28' => __( 'North Khorasan', 'lifterlms' ),
+		'29' => __( 'South Khorasan', 'lifterlms' ),
+		'30' => __( 'Alborz', 'lifterlms' ),
+		'03' => __( 'East Azerbaijan', 'lifterlms' ),
+		'07' => __( 'Fars', 'lifterlms' ),
+		'08' => __( 'Kerman', 'lifterlms' ),
+		'06' => __( 'Khuzestan', 'lifterlms' ),
+		'00' => __( 'Markazi', 'lifterlms' ),
+		'02' => __( 'Māzandarān', 'lifterlms' ),
+		'01' => __( 'Ostān-e Gīlān', 'lifterlms' ),
+		'05' => __( 'Ostān-e Kermānshāh', 'lifterlms' ),
+		'09' => __( 'Razavi Khorasan', 'lifterlms' ),
+		'04' => __( 'West Azerbaijan', 'lifterlms' ),
+	),
+	'IS' => array(
+		'1' => __( 'Capital Region', 'lifterlms' ),
+		'2' => __( 'Southern Peninsula', 'lifterlms' ),
+		'3' => __( 'West', 'lifterlms' ),
+		'4' => __( 'Westfjords', 'lifterlms' ),
+		'5' => __( 'Northwest', 'lifterlms' ),
+		'6' => __( 'Northeast', 'lifterlms' ),
+		'7' => __( 'East', 'lifterlms' ),
+		'8' => __( 'South', 'lifterlms' ),
+	),
+	'IT' => array(
+		'21' => __( 'Piedmont', 'lifterlms' ),
+		'23' => __( 'Aosta Valley', 'lifterlms' ),
+		'25' => __( 'Lombardy', 'lifterlms' ),
+		'32' => __( 'Trentino-Alto Adige', 'lifterlms' ),
+		'34' => __( 'Veneto', 'lifterlms' ),
+		'36' => __( 'Friuli Venezia Giulia', 'lifterlms' ),
+		'42' => __( 'Liguria', 'lifterlms' ),
+		'45' => __( 'Emilia-Romagna', 'lifterlms' ),
+		'52' => __( 'Tuscany', 'lifterlms' ),
+		'55' => __( 'Umbria', 'lifterlms' ),
+		'57' => __( 'The Marches', 'lifterlms' ),
+		'62' => __( 'Latium', 'lifterlms' ),
+		'65' => __( 'Abruzzo', 'lifterlms' ),
+		'67' => __( 'Molise', 'lifterlms' ),
+		'72' => __( 'Campania', 'lifterlms' ),
+		'75' => __( 'Apulia', 'lifterlms' ),
+		'77' => __( 'Basilicate', 'lifterlms' ),
+		'78' => __( 'Calabria', 'lifterlms' ),
+		'82' => __( 'Sicily', 'lifterlms' ),
+		'88' => __( 'Sardinia', 'lifterlms' ),
```

Comment:
This is a "problem".
The list here is for italian's regions.
In Italy we don't use them when specifying a billing (or even shipping) address. We use to specify what we call Provinces. Example.
I'm in Pinerolo (city/town) which is in province of Turin (province) in the Piedmont region.
Naples is both a city/town and a province in the Campania region.

When I specify my address I say:
Pinerolo (TO): TO is the italian province code for Turin.
This is the actual list we should use for italian provinces:
https://github.com/woocommerce/woocommerce/blob/e3b69b901143de3c42d90b9d4bdc97a8821133e2/i18n/states.php#L645

Maybe we should double check (compare with wc) other countries' states/provinces too...